charts/admission-controller/templates/webhook/deployment.yaml

```diff
@@ -54,8 +54,12 @@ spec:
           image: {{ include "admissionController.kspm.image" . }}
           imagePullPolicy: {{ .Values.webhook.v2.image.pullPolicy | default .Values.global.image.pullPolicy }}
           env:
+            - name: DEFAULT_TIMEOUT
+            - value: {{ .Values.webhook.timeoutSeconds}}
+            - name: DENY_ON_ERROR
```

Comment:
Should we configure the [`failurePolicy`](https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/templates/_helpers.tpl#L416) accordingly?

EG: heres what happens with the webhook right now https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/templates/_helpers.tpl#L453